pkg/actors/targets/workflow/orchestrator/orchestrator.go

```diff
@@ -80,6 +81,7 @@ type orchestrator struct {
 
 type Options struct {
 	AppID             string
+	Namespace         string
```

Comment:
Remove

Reply:
I also cannot remove this. This is 100% needed so that I know of the namespace when building out the `activityActorType = common.GetActivityActorType(o.namespace, router.GetTarget())` inside `pkg/actors/targets/workflow/orchestrator/activity.go` & `pkg/actors/targets/workflow/orchestrator/reminder.go` in the `callActivity` & `createReminder` func bc namespace is needed when constructing the ActivityActorType